README.md

```diff
@@ -179,6 +179,11 @@ Here are all the types of checks currently implemented:
 - `big_brother_service` (`Metric::BigBrotherService`): Used in conjunction with [Big Brother](https://github.com/braintree/big_brother), reports health based on the overall health of another load balanced service.
   * service
 
+- `haproxy_backends_health` (`Metric::HaproxyBackendsHealth`): Checks UP/DOWN status across a set of servers in a Haproxy cluster and computes a weight based on how many are UP.
+  * haproxy socket
+  * cluster
+  * timeout (defaults to 2s)
```

Comment:
"* weight (0-100)"

Reply:
👍  4b086b85d1e25a2d80c5e057d75e2440e8c5907d

Reply:
(We realized we should ad weight to the arguments based on this comment, so that is done ad1032455d9cdb352fa0debf01c0e3e0064a65c2)